File: app/src/main/java/bjzhou/coolapk/app/receivers/BootReceiver.java
package bjzhou.coolapk.app.receivers;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

/**
 * Created by zhoubinjia on 16/7/13.
 */
public class BootReceiver extends BroadcastReceiver {
    private static final String TAG = BootReceiver.class.getSimpleName();

    @Override
    public void onReceive(Context context, Intent intent) {
        Log.d(TAG, "onReceive: " + intent.getAction());
    }
}

File: app/src/main/kotlin/bjzhou/coolapk/app/receivers/BootReceiver.kt
package bjzhou.coolapk.app.receivers

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

/**
 * Created by zhoubinjia on 16/7/13.
 */
class BootReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        Log.d(TAG, "onReceive: " + intent.action)
    }

    companion object {
        private val TAG = BootReceiver::class.java.simpleName
    }
}
